Classes/BITCrashManager.m

```diff
@@ -548,7 +547,6 @@ - (void) unregisterObserver:(id)observer {
 - (void)leavingAppSafely {
   if (self.isAppNotTerminatingCleanlyDetectionEnabled) {
     [[NSUserDefaults standardUserDefaults] setBool:YES forKey:kBITAppWentIntoBackgroundSafely];
-    [[NSUserDefaults standardUserDefaults] synchronize];
```

Comment:
I would leave this in as one of the recommended usages of `synchronize` in the docs is:
>use this method only if you cannot wait for the automatic synchronization (for example, if your application is about to exit)